.github/ISSUE_TEMPLATE/new-extension-release.md

```diff
@@ -0,0 +1,17 @@
+---
+name: New extension release
+about: Create an issue with a checklist for the release steps (write access required
+  for the steps)
+title: Release Checklist for version xx.xx.xx
+labels: ''
+assignees: ''
+
+---
+
```

Comment:
Suggest adding "Update the version number in the issue title" as the first step?